e2e-tests/cypress/tests/integration/channels/account_settings/security/password_spec.ts

```diff
@@ -110,7 +110,7 @@ describe('Profile', () => {
         cy.get('#channelHeaderTitle').should('contain', 'Off-Topic');
     });
 
-    it('MM-T2086 Password: Timestamp and email', () => {
+    it.skip('MM-T2086 Password: Timestamp and email', () => {
```

Comment:
In the dashboard, this currently appears as "pending" rather than "skipped." I'll take a look to display the status correctly.

Reply:
Thank you!